cms/templates/base.html

```diff
@@ -56,11 +56,15 @@
 
     <a class="nav-skip" href="#main">${_("Skip to main content")}</a>
 
+    <%static:js group='base_vendor'/>
+
     <script type="text/javascript">
       window.baseUrl = "${settings.STATIC_URL | n, js_escaped_string}";
-      var require = {baseUrl: window.baseUrl};
+      require.config({
```

Comment:
So we can assume require has already been loaded (and available in the global namespace) because it is in base_vendor?